jupyterlab_git/git.py

```diff
@@ -271,7 +272,7 @@ async def changed_files(self, path, base=None, remote=None, single_commit=None):
 
         return response
 
-    async def clone(self, path, repo_url, auth=None):
+    async def clone(self, path, repo_url, meta_data_checked=False, auth=None):
```

Comment:
Let's rename the variable to match better its purpose:

```suggestion
    async def clone(self, path, repo_url, versioning=False, auth=None):
```